adage/adage/config.py.template

```diff
@@ -7,6 +7,10 @@
 # and replace with values tailored to your deployment.
 import random
 
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
@mhuyck : Can you make a proposal via edits to this branch? I don't clearly see which changes you are suggesting. I think it may be faster and more efficient for you to directly propose changes. My only preference here is to see the repository know where it needs to be placed in the minimum number of places. Thanks!